### DISCUSSION TITLE
[ERR_REQUIRE_ESM]: Must use import to load ES Module:

### DISCUSSION THREAD

**Discussion by stropicall** [avajs/ava#3050] [General]
I am getting `[ERR_REQUIRE_ESM]: Must use import to load ES Module:` while installing ava. I have `index.js` and `test.js` file and it gives this issue. I am trying to use `require` instead of `import`

```
internal/modules/cjs/loader.js:804
  throw new ERR_REQUIRE_ESM(filename);
  ^

Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /Users/bilalshah/development/projects/david/cache-test/node_modules/ava/entrypoints/cli.mjs
    at Object.Module._extensions..mjs (internal/modules/cjs/loader.js:804:9)
    at Module.load (internal/modules/cjs/loader.js:628:32)
    at Function.Module._load (internal/modules/cjs/loader.js:555:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:822:10)
    at internal/main/run_main_module.js:17:11
```

I am not much experienced. I am doing something wrong but can't figure out. Here are my test code picked from documentation and change `import` to `require`

```
const test = require('ava');

test('foo', t => {
	t.pass();
});
```

`package.json`

```
{
  "name": "cache-test",
  "main": "index.js",
  "module": "commonjs",
  "scripts": {
    "test": "ava"
  },
  "ava": {
    "extensions": [
      "js"
    ]
  },
}
```

btw this works on `node 14` and `node 16`

**Comment by novemberborn** [comment]
> btw this works on node 14 and node 16

Which version is it not working on?

**Comment by keinsell** [comment]
You have two solutions there. Downgrade `ava` to version which doesn't use Pure ESM or upgrade your project to use Pure ESM. I guess you're using `node@18` and that's why you've experienced problems.
